app/screens/user_profile/user_profile.js

```diff
@@ -197,6 +200,36 @@ export default class UserProfile extends PureComponent {
         return null;
     };
 
+    buildOrganizationBlock = () => {
+        const {theme, remoteClusterInfo} = this.props;
+        if (!remoteClusterInfo) {
+            return null;
+        }
+        const style = createStyleSheet(theme);
+        return (
+            <View>
+                <FormattedText
+                    id='mobile.routes.user_profile.organization'
+                    defaultMessage='ORGANIZATION'
+                    style={style.header}
+                />
+                <View style={style.organizationDataContainer}>
+                    <ChannelIcon
+                        isActive={true}
```

Comment:
nit
```suggestion
                        isActive
```

Reply:
All these changes are not a part of this PR.